src/main/java/Frontend/CLIHelper.java

```diff
@@ -114,12 +114,19 @@ public void trySettingOrigin(CLIHelper cliHelper, ArrayList<String> userInputs,
 
     /**
      * Simple try and catch statements to save image.
+     * @param cliHelper an instance of CLIHelper
      * @param pixels Array of integers corresponding to the colour of each pixel
+     * @param userInputs an ArrayList of strings containing user inputs
      */
-    public void trySavingImage(int[] pixels) {
+    public void trySavingImage(CLIHelper cliHelper, int[] pixels, ArrayList<String> userInputs) {
         try {
+            String name;
+            if (userInputs.contains("-save")) {
+                name = cliHelper.getCommandArgument("-save", userInputs) + ".png";
```

Comment:
Can you not call getCommandArgument directly? It seems a bit odd to have cliHelper as a parameter here

Reply:
Oh oops you're right, I was following the other codes from the file, will fix it right away along with other codes (there are multiple instances of this)